src/main/java/org/example/controllers/AdminController.java

```diff
@@ -20,12 +20,12 @@ public boolean removeDriver(@RequestParam("driverID") String driverID) {
         return adminService.removeDriver(driverID);
     }
 
-    @GetMapping(value = "/drivers/list", produces = "application/json")
+    @GetMapping("/drivers/list")
```

Comment:
Add try catch blocks here and send 500 status code to let the client know that the server failed due to some internal exception.